style.css

```diff
@@ -329,8 +461,82 @@ body.lock {
         display: none;
     }
     .content {
-        padding: 50px 0 0;
+        padding: 0 0;
     } 
+    .carousel__text {
+        top: 10%;
+        left: 3%;
+    }
+    .carousel__text1 {
+        color: var(--element-color);
+        font-size: 28px;
+        letter-spacing: 3px;
+    }
+    .carousel__text2 {
+        color: var(--additional-color);
+        font-size: 25px;
+        letter-spacing: 1.5px;
+    }
+    .carousel {
+        position: static; 
+        width: 100vw;
+        height: auto;
+        overflow: visible;
+    }
+    .carousel__box {
+        position: static;
+        display: inline;
+    }
+    .carousel__object {
+        width: 32%;
+        margin: 0 1px;
+    }
+    .carousel__buttons {
+        display: none;
+    }
+    .about-part {
+        display: flex; 
+        flex-direction: row;
+        height: auto;
+    }
+    .img1 {
+        left: 10%;
+    }
+    .img2 {
+        right: 10%; 
+    }
```

Comment:
I've seen this code already

Reply:
No, it was in mobile version with a different position (right: 0; and left: 0;), in desktop version position change to right: 10%; and left: 10%;